core/modules/file/tests/file.test

```diff
@@ -63,7 +63,7 @@ class FileTestHelper extends BackdropWebTestCase {
   protected function createFile($settings = array()) {
     // Populate defaults array.
     $settings += array(
-      'filepath' => 'Файл для тестирования ' . $this->randomName(), // Prefix with non-latin characters to ensure that all file-related tests work with international filenames.
+      'filepath' => 'Файлдлятестирования ' . $this->randomName(), // Prefix with non-latin characters to ensure that all file-related tests work with international filenames.
```

Comment:
Comments go before the line they describe.

Reply:
I have no objection to these comments :) Looks like they were there before...